.github/CODEOWNERS

```diff
@@ -12,7 +12,6 @@ backends/graphite/ @thiagoftsm @vlvkobal
 backends/json/ @thiagoftsm @vlvkobal
 backends/opentsdb/ @thiagoftsm @vlvkobal
 backends/prometheus/ @vlvkobal @thiagoftsm
-build/ @cosmix @Ferroin @knatsakis @ncmans @prologic
```

Comment:
Leave me there please

Reply:
Done :)

Reply:
This doesn't look correct. It looks like the line starting with `build/` exists twice in the file

Reply:
It only exists once; Github's diff is being strange

```sh
$ cat .github/CODEOWNERS | grep 'build\/'
build/ @cosmix
````

Reply:
It only exists once. Github's diff UI is being weird.

```sh
$ cat .github/CODEOWNERS | grep 'build\/'
build/ @cosmix
```